src/PlaywrightSharp.Tests/ElementHandleScreenshotTests.cs

```diff
@@ -417,7 +417,7 @@ public async Task ShouldTakeElementScreenshotWhenDefaultViewportIsNullAndRestore
                 <div class=""to-screenshot""></div>");
 
             var sizeBefore = await page.EvaluateAsync<ViewportSize>("() => ({ width: document.body.offsetWidth, height: document.body.offsetHeight })");
-            var elementHandle = await page.QuerySelectorAsync("div.to-screenshot");
+            await page.QuerySelectorAsync("div.to-screenshot");
             byte[] screenshot = await page.ScreenshotAsync();

```

Comment:
We should have called `elementHandle.ScreenshotAsync();` here.

Reply:
done